### ISSUE TITLE
Unblocking of ipv6 clients does not work 

### ISSUE THREAD

**Issue by stammsg** [AdguardTeam/AdGuardHome#6544]
### Prerequisites

- [X] I have checked the [Wiki](https://github.com/AdguardTeam/AdGuardHome/wiki) and [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions/categories/q-a) and found no answer

- [X] I have searched other issues and found no duplicates

- [X] I want to report a bug and not [ask a question or ask for help](https://github.com/AdguardTeam/AdGuardHome/discussions/categories/q-a)

- [X] I have set up AdGuard Home correctly and [configured clients to use it](https://github.com/AdguardTeam/AdGuardHome/wiki/Clients). (Use the [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions/categories/q-a) for help with installing and configuring clients.)

### Platform (OS and CPU architecture)

Linux, ARMv7

### Installation

GitHub releases or script from README

### Setup

On one machine

### AdGuard Home version

0.107.41 (downgraded from 0.107.42)

### Action

I have used a rule that allows all requests from an ipv4 client: 
@@||*^$client='192.168.178.24'
Now I would also like to release requests from an ipv6 client. I have created the following rule for this purpose:
@@||*^$client='fe80::5ce1:909c:9a51:4af7'
Nevertheless, the ipv6 client is not unblocked.
What am I doing wrong? How can I unblock an ipv6 client? Or is this a bug?

### Expected result

Unblocked ipv6 clients

### Actual result

Ads are still blocked...

**Comment by ainar-g** [comment]
Duplicate of #6312, related to #6192.
